TicTacIOiOS/ViewController.swift

```diff
@@ -18,32 +18,48 @@ class ViewController: UIViewController, UIAlertViewDelegate {
     @IBOutlet weak var btn7:UIButton!
     @IBOutlet weak var btn8:UIButton!
     @IBOutlet weak var label:UILabel!
-    let socket = SocketIOClient(socketURL: NSURL(string:"http://localhost:8900")!)
+    
     var name: String?
     var resetAck: SocketAckEmitter?
     let backgroundGrad = CAGradientLayer()
     
+    var inputTextField: UITextField?
+    var socket: SocketIOClient!
```

Comment:
So I looked into this. If I do it as an optional on every call to the socket I have to force unwrap it. Which I feel is more un safe. I may be missing something on how optional chaining would be an advantage. I would love to understand more.